src/components/pages/ManageResources/RenderResourceManagement.js

```diff
@@ -209,25 +215,36 @@ export const RenderResourceManagement = () => {
         </p>
         <Form.Item
           name="resource_name"
-          label="Resource Name"
+          labelWrap
+          label={
+            <label style={{ color: 'black', paddingLeft: '0rem' }}>
+              Resource
+            </label>
+          }
           value={formValues.resource_name}
           onChange={handleChange}
           rules={[{ required: true, message: 'Please input a resource name' }]}
         >
           <Input />
         </Form.Item>
         <Form.Item
-          name="category"
-          label="Category"
-          value={formValues.category}
+          name="pertains_to"
+          label={
+            <label style={{ color: 'black', padding: '0rem' }}>Category</label>
+          }
+          value={formValues.pertaians_to}
```

Comment:
Unless I'm missing something, this looks like a typo to me. It should be "pertains_to" like on line 231 without the extra "a", no?